ui/components/multichain/funding-method-modal/funding-method-modal.tsx

```diff
@@ -36,7 +37,7 @@ import {
 import { MetaMetricsContext } from '../../../contexts/metametrics';
 import FundingMethodItem from './funding-method-item';
 
-type FundingMethodModalProps = {
+type FundingMethodModalProps = Omit<ModalProps, 'children'> & {
```

Comment:
Extending ModalProps and allowing props to be passed to the root Modal element so we can pass data test id